clang/test/Sema/aarch64-incompat-sm-builtin-calls.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_cc1  -triple aarch64-none-linux-gnu -target-feature +sve \
```

Comment:
Maybe it makes more sense to move the existing `aarch64-incompat-sm-builtin-calls.c` to `aarch64-incompat-sm-builtin-calls.cpp` (there is nothing C-specific to the exisdting file AIUI) and then adding these new tests.